build/ci_utils/src/cache/goodie/graalvm.rs

```diff
@@ -90,32 +145,53 @@ impl Goodie for GraalVM {
 
 impl GraalVM {
     pub fn url(&self) -> BoxFuture<'static, Result<Url>> {
-        let platform_string = self.platform_string();
-        let graal_version_tag = self.graal_version.to_string_core();
-        let client = self.client.clone();
-        async move {
-            let repo = CE_BUILDS_REPOSITORY.handle(&client);
-            let release = repo.find_release_by_text(&graal_version_tag).await?;
-            crate::github::find_asset_url_by_text(&release, &platform_string).cloned()
+        match self.edition {
+            Edition::Community => {
+                let platform_string = self.platform_string();
+                let graal_version_tag = self.graal_version.to_string_core();
+                let client = self.client.clone();
+                async move {
+                    let repo = CE_BUILDS_REPOSITORY.handle(&client);
+                    let release = repo.find_release_by_text(&graal_version_tag).await?;
+                    crate::github::find_asset_url_by_text(&release, &platform_string).cloned()
+                }
+                .boxed()
+            },
+            Edition::Enterprise => {
+                let graal_version_tag = self.graal_version.to_string_core();
+                let url = format!(
+                    "https://download.oracle.com/graalvm/{}/archive/graalvm-jdk-{}_{}_bin.tar.gz",
+                    self.graal_version.major,
+                    graal_version_tag,
+                    self.os_arch_string(),
+                );
+                Box::pin(ready(
+                    Url::parse(&url).context("Failed to parse URL")
```

Comment:
```suggestion
                    Url::parse(&url).context("Failed to parse URL.")
```

Reply:
Done in 34923feaa7a32d4c20f93ebc54112a30f73b4ea7